projects/plugins/jetpack/modules/shortcodes/facebook.php

```diff
@@ -55,12 +55,18 @@
  * @return string Facebook embed markup.
  */
 function jetpack_facebook_embed_handler( $matches, $attr, $url ) {
+	$extra_styles = 'style="background-color: #fff; display: inline-block;"';
```

Comment:
```suggestion
	// This is a stop-gap solution until Facebook hopefully resolves this ticket
	// https://developers.facebook.com/community/threads/1675075423353415/?post_id=1675075426686748
	$extra_styles = 'style="background-color: #fff; display: inline-block;"';
```
Might be worth including a link to the issue we reported, to make it easier to understand when this can be removed.

Reply:
Good point. Done, and merging this now.